docpipeline/generate.py

```diff
@@ -392,6 +392,17 @@ def build_new_docs(bucket_name, credentials):
         new_name = blob.name[len(DOCFX_PREFIX) :]
         if new_name not in other_names:
             new_blobs.append(blob)
+        else:
+            # For existing blobs, re-build the docs if the YAML blob is newer
+            # than the existing HTML blob
+            yaml_last_updated = blob.updated
+
+            html_blob = client.bucket(bucket_name).get_blob(new_name)
```

Comment:
If you make `other_blobs` a dict instead of a list that gets turned into a set, we could avoid this `get_blob` call.

Something like this:

```python
other_blobs = {b.name: b for b in all_blobs if not b.name.startswith(DOCFX_PREFIX)}
```

Reply:
looks good. Done!